Gems/ROS2/Code/Source/VehicleDynamics/WheelDynamicsData.h

```diff
@@ -7,22 +7,24 @@
  */
 #pragma once
 
+#include <AzCore/Component/ComponentBus.h>
+#include <AzCore/Component/EntityId.h>
+
 namespace ROS2::VehicleDynamics
 {
     //! Data structure to pass wheel dynamics data for a single wheel entity.
     struct WheelDynamicsData
     {
         AZ::EntityId m_wheelEntity; //!< An entity which is expected to have a WheelControllerComponent.
-        AZ::Vector3 m_driveAxis{ AZ::Vector3::CreateAxisZ() }; //!< An axis of force application for the wheel to move forward.
+        AZ::ComponentId m_hingeJoint; //!< Steering joint
```

Comment:
AZ::ComponentId is an alias to an [AZ::u64](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/AzCore/Component/ComponentBus.h#L27), therefore it needs to be initialized as well.